ember-app/package.json

```diff
@@ -37,6 +38,8 @@
     "ember-cli-uglify": "1.0.1",
     "ember-export-application-global": "^1.0.2",
     "ember-moment": "1.1.1",
-    "ember-sinon": "0.0.3"
+    "ember-sinon": "0.0.3",
+    "rx": "^4.1.0",
```

Comment:
no, I can drop it out. 

it might have been cleaner code with RX, but I couldn't figure out how to wrap it around the plugin events, I will remove it